dev/WindowsAppRuntime_BootstrapDLL/MddBootstrapActivity.h

```diff
@@ -33,29 +36,32 @@ namespace WindowsAppRuntime::MddBootstrap::Activity
 
     class Context
     {
-        MddBootStrapAPI m_mddBootStrapAPI{ MddBootStrapAPI::None };
-        DDLMFindMethod m_ddlmFindMethodUsed{ DDLMFindMethod::None };
-
+        MddBootstrapAPI m_mddBootstrapAPI{};
         WindowsAppRuntimeBootstrap_TraceLogger::Initialize m_bootstrapInitializeActivity;
+        std::atomic<uint32_t> g_initializationCount;
+        wil::unique_cotaskmem_string g_initializationPackageFullName;
```

Comment:
Do we ever need to know the value is null (vs empty)? At a glance it seems not so this could be `std::wstring`.

Reply:
I think you had it as wil::unique_cotaskmem_string (which I maintianed) because IDynamicDependencyLifetimeManager's GetPackageFullName accepts LPWSTR and std::wstring return LPCWSTR. Do you have suggestion on elegant way to handle this ?

Reply:
Ach. OK, so mote it be.

Nothing to see here. Move along. Nothing to see ;-)